cg/test/subjects/data-shares.api.spec.js

```diff
@@ -131,3 +131,54 @@ describe('Data share remove', () => {
     expect(res.status).toEqual(NotFound.StatusCode);
   });
 });
+
+describe('Data share sharing', () => {
+  it('Should provide the users personal data in a json format if token is correct', async () => {
+    const subjectId = '2';
+    const token = await createUser(subjectId, { customData: true });
+    const res = await fetchWithAuthorization('/api/subject/data-shares/create', token, {
+      method: 'POST',
+      body: { name: 'testdatasharing' }
+    });
+    expect(res.status).toEqual(200);
+    const [dataShare] = await db('data_shares').where({ name: 'testdatasharing' });
+
+    const res2 = await fetch(`/api/subject/data-shares/share?token=${dataShare.token}`);
+
+    expect(res2.status).toEqual(200);
+    expect(await res2.json()).toEqual(expect.objectContaining({ customData: true }));
+  });
+
+  it('Should error if token is invalid', async () => {
+    const res = await fetch(`/api/subject/data-shares/share?token=badtoken`);
```

Comment:
When no method is specified, it defaults to GET right?

Reply:
correct